storage/remote/generic/generic.proto

```diff
@@ -15,9 +15,8 @@ syntax = "proto3";
 
 package generic;
 
-
 message Sample {
-  double value = 1;
+  double value       = 1;
   int64 timestamp_ms = 2;
```

Comment:
Naming seems somewhat awkward to me. What would be the overhead of having a custom timestamp message type on which we can define meaningful functions (like getting a `time.Time` etc.)?